.mvn/maven.config

```diff
@@ -1 +1,2 @@
 -Pbuild-individual-bundles
+-Dtycho.target.eager=true
```

Comment:
Sorry if stupid question , but isn't that the opposite of PR message?

Reply:
Commit message was wrong, fixed that.